buildrunner/steprunner/tasks/build.py

```diff
@@ -204,15 +207,29 @@ def run(self, context):
         buildrunner_config = BuildRunnerConfig.get_instance()
         docker_registry = buildrunner_config.global_config.docker_registry
         self.step_runner.log.write("Running docker build\n")
-        builder = DockerBuilder(
-            self.path,
-            inject=self.to_inject,
-            dockerfile=self.dockerfile,
-            docker_registry=docker_registry,
-        )
+
         try:
-            if self.platforms:
-                built_image = self.step_runner.multi_platform.build_multiple_images(
+            if self.platforms or not buildrunner_config.run_config.use_legacy_builder:
+                if self.platforms and buildrunner_config.run_config.use_legacy_builder:
+                    LOGGER.warning(
+                        f"Ignoring use-legacy-builder. Using the legacy builder for multiplatform images {self.platforms} is not supported. "
+                        "If you are want to use the legacy builder and specify the platform, please use either 'platform'."
```

Comment:
`either platform`?

Reply:
Nice catch! I think I was going to say "use either 'platform' or 'platforms' with a single platform". But I decided not to allow `platforms` at all with the legacy builder. But I didn't correct the text here. I fixed it now.